### PR TITLE
Update 1.1

### PR DESCRIPTION
**Pull request by Rainb0wKey**
Usage > Syntax
emojis;... > emojis

`### Parameters > ## Parameters`

### CODE REVIEW

File: src/bdscript/addCmdReactions.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # $addCmdReactions
 Adds reactions to the message that triggered the command.
 
-## Usage
+## Syntax
 ```
-$addCmdReactions[emojis;...]
+$addCmdReactions[emojis]
```
Comment (MineBartekSA) [inline src/bdscript/addCmdReactions.md]: This ellipsis is important. It quickly shows that you can repeat the last argument.
```suggestion
$addCmdReactions[emojis;...]
```
At the least, this is the reason it's there in the first place.

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Comment by Rainb0wKey** [comment]
What about $addReactions?

**Comment by Rainb0wKey** [comment]
> What about $addReactions?

Updated^

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**